cmd/alerts/silence/common.go

```diff
@@ -15,14 +15,35 @@ const (
 	AccountNamespace = "openshift-monitoring"
 	ContainerName    = "alertmanager"
 	LocalHostUrl     = "http://localhost:9093"
-	PodName          = "alertmanager-main-0"
+	PrimaryPod       = "alertmanager-main-0"
+	SecondaryPod     = "alertmanager-main-1"
 )
 
+// ExecInPod is designed to execute a command inside a Kubernetes pod and capture its output.
 func ExecInPod(kubeconfig *rest.Config, clientset *kubernetes.Clientset, cmd []string) (string, error) {
+	var cmdOutput string
+	var err error
 
-	req := clientset.CoreV1().RESTClient().Post().Resource("pods").Name(PodName).
-		Namespace(AccountNamespace).SubResource("exec")
+	// Attempt to execute with the primary pod
+	cmdOutput, err = ExecWithPod(kubeconfig, clientset, PrimaryPod, cmd)
+	if err == nil {
+		return cmdOutput, nil // Successfully executed
+	}
 
+	// If execution with primary pod fails, try with the secondary pod
```

Comment:
Instead of this comment, can we actually output a log here that says something similar? i.e. "Execution with alertmanager-main-0 failed - attempting with alertmanger-main-1" or something similar.